app/src/main/java/org/mozilla/fenix/wallpapers/Wallpaper.kt

```diff
@@ -74,7 +74,29 @@ data class Wallpaper(
          * @param theme One of dark/light.
          * @param name The name of the wallpaper.
          */
-        fun getBaseLocalPath(orientation: String, theme: String, name: String): String =
+        fun legacyGetLocalPath(orientation: String, theme: String, name: String): String =
             "wallpapers/$orientation/$theme/$name.png"
+
+        /**
+         * Defines the standard path at which a wallpaper resource is kept on disk.
+         *
+         * @param type The type of image that should be retrieved.
+         * @param name The name of the wallpaper.
+         */
+        fun getLocalPath(type: ImageType, name: String) = "wallpapers/$name/${type.lowercase()}.png"
+    }
+
+    /**
+     * Defines various image asset types that can be downloaded for each wallpaper.
+     */
+    enum class ImageType {
+        Portrait,
+        Landscape,
+        Thumbnail;
```

Comment:
enum class types are typically all uppercase

Reply:
Lemme preface this with saying that I am definitely not willing to die on this hill so am happy to change it if you'd like. First though, I have a rant prepared

`enum class`es were required to be uppercase in Java originally (and maybe still?). Kotlin, however, does not require it. Uppercasing them is a stylistic choice that is largely a holdover from the Java days. Even in our codebase, there are mixed usages of the two styles. For example, `BrowserDirections` is pascal cased but `HomeScreenSection` is upper cased. Unless there's a distinction between the two for a reason I'm unaware of, I just tend to prefer not getting yelled at by my code lol

Reply:
lol well it's not a merge blocker, for sure. I didn't realize there was both uppercase and mixed case in our codebase, so maybe someone has some context for that.

Reply:
I think the mix we currently have is because Kotlin does not enforce uppercase and allows for either naming scheme [depending on an abstract concept of "usage"](https://kotlinlang.org/docs/coding-conventions.html#property-names):
> For enum constants, it's OK to use either uppercase underscore-separated names ([screaming snake case](https://en.wikipedia.org/wiki/Snake_case)) (enum class Color { RED, GREEN }) or upper camel case names, depending on the usage.

We could look into creating our own coding style guide to eliminate subjectivity, have the code more cohesive and simpler reviews.